docs/docs/reference/enums/desugarEnums.md

```diff
@@ -158,29 +158,28 @@ map into `case class`es or `val`s.
 
 ### Translation of Enumerations
 
-Non-generic enums `E` that define one or more singleton cases
-are called _enumerations_. Companion objects of enumerations define
-the following additional synthetic members.
+An enum `E` (possibly generic) that defines one or more singleton cases
+will define the following additional synthetic members in its companion object (where `E'` denotes `E` with
+any type parameters replaced by wildcards):
 
-   - A method `valueOf(name: String): E`. It returns the singleton case value whose
+   - A method `valueOf(name: String): E'`. It returns the singleton case value whose
      `toString` representation is `name`.
-   - A method `values` which returns an `Array[E]` of all singleton case
-     values in `E`, in the order of their definitions.
+   - A method `values` which returns an `Array[E']` of all singleton case
+     values defined by `E`, in the order of their definitions.
 
-Companion objects of enumerations that contain at least one simple case define in addition:
+Companion objects of enumerations (i.e. `E` is not generic) that contain at least one simple case define in addition:
```

Comment:
This whole section is called "Translation of Enumerations" and covers both generic and non-generic, so non-generic enumeration need a more specialized name than "enumerations" (e.g., "monomorphic enumeration", or "simple enum")

Reply:
Ah yeah that was an oversight, I've got rid of any mention of "enumeration" now, as that was previously defined as requiring "non-generic" base class